### ISSUE TITLE
Select Frontend Framework & Libraries

### ISSUE THREAD

**Issue by superstructor** [clj-sg/clj-sg-website#5]
e.g.

- [rum](https://github.com/tonsky/rum)
- [citrus](https://github.com/roman01la/citrus)
- [re-frame](https://github.com/Day8/re-frame)
- [reagent](https://github.com/reagent-project/reagent)
- [om](https://github.com/omcljs/om)
- [garden](https://github.com/noprompt/garden)

**Comment by RohitTulu** [comment]
Between Garden and Sass, I would choose garden, because of its more closer connection to the Clojure programming language. Both of the technologies are powerful however, Sass has additional dependencies like ruby or libsass. Garden is neat and powerful because we have the entire clojure stack in the backend.

Having researched on Om Next, it is said to be quite powerful, because of its richer community, however, Reframe which is built on top of ReAgent is more intuitive as it separates the concerns. Hence, I would like to use reframe to build the UI.

I spent some time on run, however, I found it a little difficult to work with and it is very closely tied to the react component lifecycle.  Citrus is just a state manager for rum, which was named as Scrum once.
